File: task.kt
fun isPangram(s: String): String {
    val letters = BooleanArray(26)
    var count = 0
    
    for (c in s.lowercase()) {
        if (c in 'a'..'z') {
            val index = c - 'a'
            if (!letters[index]) {
                letters[index] = true
                count++
            }
        }
    }
    
    return if (count == 26) "Yes" else "No"
}

fun main() {
    print("Введите строку: ")
    val input = readLine() ?: ""
    println(isPangram(input))
}

File: task.exs
defmodule Pangram do
  def is_pangram(s) do
    unique_count = s
    |> String.downcase()
    |> String.replace(~r/[^a-z]/, "")
    |> String.graphemes()
    |> Enum.uniq()
    |> Enum.count()
    
    if unique_count == 26 do
      "Yes"
    else
      "No"
    end
  end
end

IO.puts("Введите строку: ")
input = IO.gets("") |> String.trim()
IO.puts(Pangram.is_pangram(input))

